bin/setup-local-env.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Exit if any command fails
+set -e
+
+# Change to the expected directory
+cd "$(dirname "$0")/../docker"
+
+# Launch the WordPress docker
+docker-compose up -d
+
+# Wait until the docker containers are setup properely
+echo "Attempting to connect to wordpress"
+until $(curl -L http://localhost:8888 -so - | grep -q "WordPress"); do
```

Comment:
Minor: 8888 is a fairly common port, so we could encounter a conflict; then again, the risk always exists.

Reply:
Is `1440` an option? It would align nicely with `Printing since 1440.` 💯